packages/react-devtools-shared/src/backend/agent.js

```diff
@@ -148,6 +148,7 @@ export default class Agent extends EventEmitter<{
   stopInspectingNative: [],
   shutdown: [],
   traceUpdates: [Set<NativeType>],
+  storeConsolePatchSettings: ConsolePatchSettings,
```

Comment:
Are we using this in this PR? If not maybe we can colocate this in a PR where we're actually using this code

Reply:
great catch, its unused

Reply:
good catch, its unused, will remove